src/blog/2024/05/flowfuse-on-robustel-eg5120.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Run FlowFuse on robustel EG5120
+subtitle: How to install FlowFuse Device Agent on robustel EG5120
+description: In this guide, we will discuss how to install FlowFuse Device agent on robustel EG5120.
+date: 2024-05-01
+authors: ["grey-dziuba"]
+image: /blog/2024/05/images/flowfuse-on-rubustel-eg5120-node-red.png
+tags:
+    - posts
+    - flowfuse
+    - node.js
+    - robustel
+    - eg5120
+---
+
+The [robustel EG5120](https://www.robustel.com/product/eg5120-industrial-edge-computing-gateway/) is a versatile gateway that facilitates robust connectivity for industrial IoT applications. Integrating this powerful hardware with FlowFuse not only enhances its capabilities but also simplifies the management and deployment process. In this guide, we’ll walk through the steps to integrate the robustel EG5120 with FlowFuse, making your IoT solutions more efficient and easier to manage.
+
+<!--more-->
+
+#### **Introduction**
+
+This blog is designed for IoT enthusiasts and professionals who are looking to enhance their industrial IoT applications. The [robustel EG5120](https://www.robustel.com/product/eg5120-industrial-edge-computing-gateway/), equipped with Linux-based Debian 11 and preinstalled Node-RED, offers robust connectivity options. When combined with FlowFuse, this gateway becomes even more powerful, enabling seamless device management and deployment. 
+
+
+#### **Robustel and Node-RED**
```

Comment:
`##` means H2, which is I think what we want and has better styling so no need to make the title bold

```suggestion
## Robustel and Node-RED
```